src/app/auth/change-password/change-password.tsx

```diff
@@ -0,0 +1,88 @@
+'use client'
+
+import ChangePasswordSchema from '@/app/auth/change-password/change-password-schema'
+import { useForm } from 'react-hook-form'
+import { z } from 'zod'
+import { zodResolver } from '@hookform/resolvers/zod'
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form'
+import { Input } from '@/components/ui/input'
+import { Button } from '@/components/ui/button'
+import AuthCard from '@/app/auth/components/auth-card'
+import Link from 'next/link'
+
+export function ChangePasswordForm() {
+  const form = useForm<z.infer<typeof ChangePasswordSchema>>({
+    resolver: zodResolver(ChangePasswordSchema),
+    defaultValues: {
+      password: '',
+      confirmPassword: '',
+    },
+  })
+
+  function onSubmit(values: z.infer<typeof ChangePasswordSchema>) {
```

Comment:
Use server actions. Refer to login page implementation

```suggestion
const [isPending, startTransition] = useTransition()

  const onSubmit: SubmitHandler<z.infer<typeof LoginSchema>> = (data) => {
    startTransition(() => loginAction(data))
  }
```